docs/metrics.md

```diff
@@ -92,4 +92,26 @@ def train_model(...):
             n_invalid_images,
             metric_type=MetricType.COUNT
         )
-```
\ No newline at end of file
+```
+
+#### Run vs Pipeline Metrics
+
+You can create your own pipeline-level metrics to track how a pipeline's results are changing over time. To do this,
+use the `scope` parameter of `log_metric`:
+
+```python
+from sematic.ee.metrics import log_metric, MetricType, MetricScope
+
+@sematic.func
+def eval_model(...):
+    # ...
+    log_metric(
+        "accuracy",
+        accuracy,
+        metric_type=MetricType.GAUGE,
+        scope=MetricScope.PIPELINE,
+    )
+```
+
+Instead of appearing in the "Metrics" tab of a run's view, pipeline-level metrics will appear in the
+metric details view obtained by clicking on the pipeline summary metrics (run count, success rate, etc).
```

Comment:
Add screenshot?

Reply:
I don't have ref to a pipeline with a relevant screenshot at the moment, and it would take several pipeline executions and some planning to make one that looked real and good 😆